README.md

```diff
@@ -1,2 +1,33 @@
-# interpret-iqms
-Interpreting the IQMs of MRIQC
+# Interpreting the IQMs of MRIQC
+
+In this project we want to improve the interpretability of the Image Quality Metrics (IQMs) generated by MRIQC.
+
+**What is MRIQC?**
+
+MRIQC is an open-source tool to assess the quality of MRI images. It generates visual reports for every scan, as well as a number of IQMs which are collected in a group-level report.
```

Comment:
```suggestion
MRIQC is an open-source tool to assess the quality of structural and functional MRI images. It generates visual reports for every scan, as well as a number of IQMs which are collected in a group-level report.
```
I would precise because as for now diffusion is not supported by MRIQC